101-DDoS-Attack-Prevention/DSC/Configure-AzureSecurityCenter.ps1

```diff
@@ -0,0 +1,87 @@
+
+[CmdletBinding()]
+param (
+
+    [Parameter(Mandatory = $true,
+        HelpMessage = "Provide email address for recieving alerts from Azure Security Center.")]
+    [Alias("email")]
+    [string]
+    $EmailAddressForAlerts
+
+)
+
+# Enable ASC Policies
+
+$azureRmProfile = [Microsoft.Azure.Commands.Common.Authentication.Abstractions.AzureRmProfileProvider]::Instance.Profile
+Write-Verbose "Checking AzureRM context for Azure security center configuration."
+$currentAzureContext = Get-AzureRmContext
+$profileClient = New-Object Microsoft.Azure.Commands.ResourceManager.Common.RMProfileClient($azureRmProfile)
+
+Write-Verbose "Getting access token for Azure security center."
+Write-Verbose("Getting access token for tenant" + $currentAzureContext.Subscription.TenantId)
+$token = $profileClient.AcquireAccessToken($currentAzureContext.Subscription.TenantId)
+$token = $token.AccessToken
+$Script:asc_clientId = "1950a258-227b-4e31-a9cf-717495945fc2"              # Well-known client ID for Azure PowerShell
+$Script:asc_redirectUri = "urn:ietf:wg:oauth:2.0:oob"                      # Redirect URI for Azure PowerShell
+$Script:asc_resourceAppIdURI = "https://management.azure.com/"             # Resource URI for REST API
+$Script:asc_url = 'management.azure.com'                                   # Well-known URL endpoint
+$Script:asc_version = "2015-06-01-preview"                                 # Default API Version
```

Comment:
update apiVersion

Reply:
the latest one version 2017-01-01-alpha does not support policy object it in. got a few errors.